addons/html_builder/static/src/builder/plugins/setup_editor_plugin.js

```diff
@@ -24,7 +26,13 @@ export class SetupEditorPlugin extends Plugin {
             .filter((el) => !el.hasAttribute("data-oe-sanitize-prevent-edition"));
         editableEls.concat(Array.from(this.editable.querySelectorAll(".o_editable")));
         editableEls.forEach((el) => el.classList.add("o_editable"));
+        this.beforeUnloadFc = this.addDomListener(window, "beforeunload", (event) => {
+            if (this.isDirty()) {
+                event.returnValue = _t("This document is not saved!");
```

Comment:
document ? It s a strange name. Maybe "This website page is not saved!" ? Maybe check with lionel